From 0f31b60543147a424945b687ef70f1d0c3d5aeec Mon Sep 17 00:00:00 2001
From: Sooyong Suk <s.suk@samsung.corp-partner.google.com>
Date: Fri, 5 Jun 2026 18:58:31 +0900
Subject: [PATCH 207/223] ANDROID: GKI: update symbol list for galaxy

1 function symbol(s) added
  'int __traceiter_android_rvh_do_swap_page_start(void*, swp_entry_t)'

1 variable symbol(s) added
  'struct tracepoint __tracepoint_android_rvh_do_swap_page_start'

Bug: 514180992
Bug: 521571739
Change-Id: Ia0c76708bde48314f94359b30f95cb35ad6bace0
(cherry picked from commit cb3d3d97558cf70b7557afe2ada7994268928fa0)
Signed-off-by: Sooyong Suk <s.suk@samsung.corp-partner.google.com>
---
 gki/aarch64/abi.stg        | 20 ++++++++++++++++++++
 gki/aarch64/symbols/galaxy |  2 ++
 2 files changed, 22 insertions(+)

diff --git a/gki/aarch64/abi.stg b/gki/aarch64/abi.stg
index 56295efbe0dd..2511c9783c5f 100644
--- a/gki/aarch64/abi.stg
+++ b/gki/aarch64/abi.stg
@@ -440541,6 +440541,15 @@ elf_symbol {
   type_id: 0x9b379132
   full_name: "__traceiter_android_rvh_do_swap_page_relax"
 }
+elf_symbol {
+  id: 0x970f3917
+  name: "__traceiter_android_rvh_do_swap_page_start"
+  is_defined: true
+  symbol_type: FUNCTION
+  crc: 0xba7470ba
+  type_id: 0x9b33e2c9
+  full_name: "__traceiter_android_rvh_do_swap_page_start"
+}
 elf_symbol {
   id: 0x255d03b3
   name: "__traceiter_android_rvh_dpm_prepare"
@@ -448605,6 +448614,15 @@ elf_symbol {
   type_id: 0x18ccbd2c
   full_name: "__tracepoint_android_rvh_do_swap_page_relax"
 }
+elf_symbol {
+  id: 0x70d6de89
+  name: "__tracepoint_android_rvh_do_swap_page_start"
+  is_defined: true
+  symbol_type: OBJECT
+  crc: 0x7c5aa8a7
+  type_id: 0x18ccbd2c
+  full_name: "__tracepoint_android_rvh_do_swap_page_start"
+}
 elf_symbol {
   id: 0x6de60f21
   name: "__tracepoint_android_rvh_dpm_prepare"
@@ -528209,6 +528227,7 @@ interface {
   symbol_id: 0x611114e0
   symbol_id: 0x76eb11ee
   symbol_id: 0x8d7abadb
+  symbol_id: 0x970f3917
   symbol_id: 0x255d03b3
   symbol_id: 0x6682daab
   symbol_id: 0x24dfaeb0
@@ -529105,6 +529124,7 @@ interface {
   symbol_id: 0xcfcbb846
   symbol_id: 0x1d7748c4
   symbol_id: 0x17ec58c1
+  symbol_id: 0x70d6de89
   symbol_id: 0x6de60f21
   symbol_id: 0xf9b36d15
   symbol_id: 0xbade8646
diff --git a/gki/aarch64/symbols/galaxy b/gki/aarch64/symbols/galaxy
index e4fbbdc63463..7c8bac0a59f8 100644
--- a/gki/aarch64/symbols/galaxy
+++ b/gki/aarch64/symbols/galaxy
@@ -54,6 +54,7 @@
   __traceiter_android_rvh_do_sea
   __traceiter_android_rvh_do_sp_pc_abort
   __traceiter_android_rvh_do_swap_page_relax
+  __traceiter_android_rvh_do_swap_page_start
   __traceiter_android_rvh_handle_bad_stack
   __traceiter_android_rvh_panic_unhandled
   __traceiter_android_rvh_process_madvise_bypass
@@ -144,6 +145,7 @@
   __tracepoint_android_rvh_do_sea
   __tracepoint_android_rvh_do_sp_pc_abort
   __tracepoint_android_rvh_do_swap_page_relax
+  __tracepoint_android_rvh_do_swap_page_start
   __tracepoint_android_rvh_handle_bad_stack
   __tracepoint_android_rvh_panic_unhandled
   __tracepoint_android_rvh_process_madvise_bypass
-- 
2.50.1

